components/tools/ToolsCard.js

```diff
@@ -0,0 +1,107 @@
+import { useState, useEffect } from 'react'
+import Heading from '../typography/Heading';
+import Paragraph from '../typography/Paragraph';
+import Tag from './Tags';
+import TextTruncate from 'react-text-truncate';
+
+export default function toolsCard({ toolData }) {
+  const [showDescription, setShowDescription] = useState(false)
+  const [onGithub, setonGithub] = useState(false)
+
+  useEffect(() => {
+    const url = new URL(toolData.links.repoUrl)
+    if (url.host == 'github.com') setonGithub(true)
+    else setonGithub(false)
```

Comment:
You don't need to make it as useEffect, you can perform logic in separate function, or even outside that useEffect - and remove `const [onGithub, setOnGithub] = useState(false)` useState.